cogs/images.py

```diff
@@ -75,17 +76,18 @@ class Images(commands.Cog):
 
         EXAMPLE: !horse
         RESULT: A beautiful picture of a horse o3o"""
-        result = await utils.request("http://hrsendl.com/horse")
+        data = await utils.request("http://hrsendl.com/api/horse")
+        result = data['data']
```

Comment:
This should be done after the if statement, and the if statement should be changed to check `data` since that's checking if there was some network issue and it couldn't connect at all.